Source/WebCore/accessibility/atspi/AccessibilityAtspi.cpp

```diff
@@ -288,7 +288,7 @@ void AccessibilityAtspi::registerRoot(AccessibilityRootAtspi& rootObject, Vector
     }
 
     ensureCache();
-    String path = makeString("/org/a11y/webkit/accessible/", createVersion4UUIDString().replace('-', '_'));
+    String path = makeString("/org/a11y/webkit/accessible/", makeStringByReplacingAll(createVersion4UUIDString(), '-', '_'));
```

Comment:
These are the cases that would benefit from an in-place StringConcatenate manipulator that would replace characters during the string building process. By the way, Chris, I’m not asking you to personally build all these, just talking about the repertoire we’d like to have.